syft/frameworks/torch/hook.py

```diff
@@ -122,7 +122,12 @@ def __init__(
         # Add all hooked tensor methods to pointer but change behaviour to have the cmd sent
         self._hook_pointer_tensor_methods()
 
-        # Add all hooked tensor methods to multi_pointer to change behavior to hav ethe cmd
+        # Add all hooked tensor methods to AdditiveSharingTensor tensor but change behaviour
+        # to all shares (when it makes sense, otherwise the method is overwritten in the
+        # AdditiveSharingTensor class)
+        self._hook_additive_shared_tensor_methods()
```

Comment:
By default yes, and we rewrite all methods taht we need and which and not correctly hooked - this could be discussed. (I tried discussing this in #1906 ;) )